src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -98,6 +96,10 @@ void GcsPlacementGroupScheduler::ScheduleUnplacedBundles(
                 .emplace(placement_group->GetPlacementGroupID(), lease_status_tracker)
                 .second);
 
+  // Acquire resources from gcs resources manager to reserve bundle resources.
+  const auto &prepared_bundle_locations = lease_status_tracker->GetBundleLocations();
```

Comment:
Can you change the variable name to `bundle_locations`? I think `prepared_bundle_locations ` is incorrect because at this time the bundles are not prepared yet